config.py

```diff
@@ -35,6 +35,8 @@ class Config(object):
     ACCOUNT_SERVICE_URL = os.getenv("ACCOUNT_SERVICE_URL")
     ACCOUNT_SERVICE_LOG_OUT_URL = os.getenv("ACCOUNT_SERVICE_LOG_OUT_URL")
     EQ_URL = os.getenv("EQ_URL")
+    EQ_V3_URL = EQ_URL
+    # EQ_V3_URL = os.getenv("EQ_V3_URL") TODO uncomment and make this work for real
```

Comment:
We should be doing this as a feature flag rather than relying on us remembering to uncomment things